tests/conftest.py

```diff
@@ -24,7 +24,7 @@
     OpType,
 )
 
-from pytket_qir.qir import write_qir_file
+from pytket_qir.qir import write_qir_file  # type: ignore
```

Comment:
Is this (type ignore) necessary?

Reply:
Hm, at some point, `mypy` on CI complained about it. Will double check.